packages/docs/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@vuestic-ui/docs",
+  "version": "0.1.8",
+  "description": "Documentation for vuestic-ui framework",
+  "main": "index.js",
+  "repository": "https://github.com/epicmaxco/vuestic-ui",
+  "license": "MIT",
+  "dependencies": {
+    "core-js": "2",
```

Comment:
Just 2? o.o

Reply:
It is recommended in official docs to use v2, not 3. 3 causes module errors.